packages/webview_flutter/webview_flutter_platform_interface/test/src/method_channel/webview_method_channel_test.dart

```diff
@@ -73,6 +90,52 @@ void main() {
       );
     });
 
+    test('loadFile with invalid file', () async {
+      expect(
+        () => webViewPlatform.loadFile('invalid file'),
+        throwsA(
+          isA<ArgumentError>().having(
+            (ArgumentError error) => error.message,
+            'message',
+            'Failed loading file.',
+          ),
+        ),
+      );
+    });
+
+    test('loadFlutterAsset', () async {
+      await webViewPlatform.loadFlutterAsset(
+        'folder/asset.html',
+      );
+
+      expect(
+        log,
+        <Matcher>[
+          isMethodCall(
+            'loadFlutterAsset',
+            arguments: 'folder/asset.html',
+          ),
+        ],
+      );
+    });
+
+    test('loadFlutterAsset with empty key', () async {
+      expect(() => webViewPlatform.loadFlutterAsset(''), throwsAssertionError);
+    });
+
+    test('loadFlutterAsset with invalid key', () async {
+      expect(
+        () => webViewPlatform.loadFlutterAsset('invalid key'),
+        throwsA(
+          isA<ArgumentError>().having(
+            (ArgumentError error) => error.message,
+            'message',
+            'Failed loading asset.',
+          ),
+        ),
+      );
+    });
```

Comment:
Please also add tests for both that errors with different codes aren't remapped to `ArgumentError`s.